keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -470,9 +490,7 @@ def from_preset(
 
         config = metadata["preprocessor_config"]
         config.update(
-            {
-                "vocabulary": vocabulary,
-            },
+            {"vocabulary": vocabulary, "vocabulary_size": len(vocabulary)},
```

Comment:
I know it's okay if we pass `len(vocabulary)` here...but ideally, should config parameter values not be the same as the passed argument to `__init__`? So, if `None` is passed for `vocabulary_size`, shouldn't it be `None` here?

Reply:
Yes, that makes sense I'm changing it.